todo_app/templates/item_edit.html

```diff
@@ -0,0 +1,51 @@
+{% extends "layout.html" %}
+{% block title %}Editing {{ item.title }}{% endblock %}
+
+{% block content %}
+  <div class="row">
+    <div class="col-md-12">
+      <h2>Editing {{ item.title }}</h2>
+      <form action="/items" id="editForm">
+        <input type="hidden" name="id" id="id" value="{{ item.id }}">
+        <label for="title">Title: </label>
+        <input type="text" name="title" id="title" placeholder="{{ item.title }}">
+        <br>
+        <label for="listId">List: </label>
+        <select name="listId" id="listId">
+          {% for lst in lists %}
```

Comment:
Very minor, but I'd always be inclined to spell out `list` here - it's one extra character but easier to parse for someone not expecting the abbreviation

Reply:
I've raised an issue to track this.